rpc/manager/manager.proto

```diff
@@ -170,6 +170,7 @@ message IngressInfo {
 message PreviewSpec {
   IngressInfo ingress = 1;
   bool display_banner = 2;
+  map<string,string> inject_headers = 3;
```

Comment:
Taking a lesson from Emissary's `add_request_headers`, it seems likely to me that at some point we will want to have extra instructions associated with the value; like "append vs overwrite". Even if we don't want to implement that right now, it seems wise to me to leave that possibility open; as

```protobuf
message InjectedHeader {
    string value = 1;
}
map<string,InjectedHeader> inject_headers = 3;
```